index.html

```diff
@@ -16,8 +16,19 @@
       },
       {
         name: "Yuri Shkuro",
-        company: "Facebook",
+        company: "Meta",
         w3cid: "104074"
+      },
+      {
+        name: "Daniel Dyla",
+        company: "Dynatrace",
+        companyURL: "https://dynatrace.com",
+        w3cid: "117848"
+      },
+      {
+        name: "J. Kalyana Sundaram",
+        company: "Microsoft",
+        w3cid: "??????"
       }],
       formerEditors:  [{
```

Comment:
@plehegar should the former employer be preserved for the former editors section?

Reply:
moving former editors to acknowledgements section will eliminate the problem as we mention people by full name, not the employer

Reply:
That's being taken care of since we removed companies from former editors.